app/models/reservation.rb

```diff
@@ -77,7 +80,10 @@ def status # rubocop:disable CyclomaticComplexity, PerceivedComplexity
         '?'
       end
     elsif checked_in.nil?
-      due_date < Time.zone.today ? 'overdue' : 'checked out'
+      if due_date.midnight == Date.current.midnight then 'due today'
```

Comment:
This doesn't need the `.midnight` method, you can just do `due_date == Time.zone.today`